README.md

```diff
@@ -26,7 +26,7 @@ Plugin:
 ```shell
 asdf plugin add ansible
 # or
-asdf plugin add https://github.com/Bleacks/asdf-ansible.git
+asdf plugin add ansible https://github.com/Bleacks/asdf-ansible.git
```

Comment:
Repo name not in synch...

asdf plugin add ansible https://github.com/Bleacks/asdf-ansible-plugin.git

Reply:
neither is the file I modified, in that case.   in fact pretty much every reference to the repo in the master branch is wrong.  that's probably a PR for someone else to fix. :-D